tests/test_preupg.py

```diff
@@ -273,95 +274,125 @@ def test_solution_www_bold_tag(self):
 
 
 class TestScenario(base.TestCase):
-    temp_dir = None
-
-    def setUp(self):
-        self.temp_dir = tempfile.mkdtemp()
-
-    def tearDown(self):
-        shutil.rmtree(self.temp_dir)
-
+    '''
+    Test get_scenario method, to get directory with modules
+    '''
     def test_correct_content_scenario(self):
```

Comment:
Rename to use the current terminology in which content=>module, scenario=module set.